src/Livewire/Accounting/PaymentReminder.php

```diff
@@ -53,30 +60,114 @@ protected function getSelectedActions(): array
             DataTableButton::make()
                 ->color('primary')
                 ->label(__('Create Payment Reminder'))
-                ->wireClick('createNextPaymentReminder'),
+                ->wireClick('openCreateDocumentsModal'),
+        ];
+    }
+
+    protected function getAttachments(OffersPrinting $item): array
+    {
+        return [
+            'name' => $item->order->invoice()->file_name,
+            'id' => $item->order->invoice()->id,
+        ];
+    }
+
+    protected function getCommunicatableType(OffersPrinting $item): string
+    {
+        return morph_alias(Order::class);
+    }
+
+    protected function getCommunicatableId(OffersPrinting $item): int
+    {
+        return $item->order_id;
+    }
+
+    protected function getTo(OffersPrinting $item, array $documents): array
+    {
+        return Arr::wrap($item->getPaymentReminderText()?->mail_to
+            ?: $item->order->contact->invoiceAddress->email_primary
+        );
+    }
+
+    protected function getCc(OffersPrinting $item): array
+    {
+        return Arr::wrap($item->getPaymentReminderText()?->mail_cc);
+    }
+
+    protected function getCreateAttachmentArray(OffersPrinting $item, string $view): array
+    {
+        return [
+            'model_type' => $item->getMorphClass(),
+            'model_id' => $item->getKey(),
+            'view' => $view,
+            'name' => __($view),
+            'attach_relation' => 'order',
         ];
     }
 
-    public function createNextPaymentReminder(): void
+    protected function getSubject(OffersPrinting $item): string
+    {
+        return html_entity_decode($item->getPaymentReminderText()?->mail_subject ?? '') ?:
+            __(
+                'Payment Reminder :level for invoice :invoice-number',
```

Comment:
please update localization key to :invoice_number to keep it consistent.

Reply:
√